.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,15 @@
-> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.
+> You may disregard these recommendations if you used the **Edit on GitHub**
```

Comment:
don't include .github - there is a reason for this file to be formatted this way (long single line) dues to the way Github handles them

Reply:
Added `.github` folder to `.prettierignore` and returned back old formatting